homeassistant/components/wiz_light/light.py

```diff
@@ -0,0 +1,381 @@
+        await self._light.turn_off()
+
+    @property
+    def color_temp(self):
+        """Return the CT color value in mireds."""
+        return self._temperature
+
+    def get_min_mireds(self) -> int:
+        """Return the coldest color_temp that this light supports."""
+        if self._bulbtype is None:
+            return color_utils.color_temperature_kelvin_to_mired(6500)
+        if self._bulbtype.bulb_type != BulbClass.DW:
+            # If bulbtype is TW or RGB then return the kelvin value
+            try:
+                return color_utils.color_temperature_kelvin_to_mired(
+                    self._bulbtype.kelvin_range.max
+                )
+
+            except WizLightNotKnownBulb:
+                _LOGGER.info("Kelvin is not present in the library. Fallback to 6500")
+                return color_utils.color_temperature_kelvin_to_mired(6500)
+        else:
+            # DW bulbs have a fixed range of color temperature
+            return 0
+
+    def get_max_mireds(self) -> int:
+        """Return the warmest color_temp that this light supports."""
+        if self._bulbtype is None:
+            return color_utils.color_temperature_kelvin_to_mired(2200)
+        if self._bulbtype.bulb_type != BulbClass.DW:
+            # If bulbtype is TW or RGB then return the kelvin value
+            try:
+                return color_utils.color_temperature_kelvin_to_mired(
+                    self._bulbtype.kelvin_range.min
+                )
+            except WizLightNotKnownBulb:
+                _LOGGER.info("Kelvin is not present in the library. Fallback to 2200")
+                return color_utils.color_temperature_kelvin_to_mired(2200)
+        else:
+            # DW bulbs have a fixed range of color temperature
+            return 0
+
+    def get_supported_features(self) -> int:
+        """Flag supported features."""
+        if self._bulbtype:
+            return self.featuremap()
+        # fallback
+        return SUPPORT_FEATURES_RGB
+
+    @property
+    def effect(self):
+        """Return the current effect."""
+        return self._effect
+
+    @property
+    def effect_list(self):
+        """Return the list of supported effects.
+
+        URL: https://docs.pro.wizconnected.com/#light-modes
+        """
+        return self._scenes
+
+    @property
+    def available(self):
+        """Return if light is available."""
+        return self._available
+
+    async def async_update(self):
+        """Fetch new state data for this light."""
+        await self.update_state()
+
+        if self._state is not None and self._state is not False:
+            self.update_brightness()
+            self.update_temperature()
+            self.update_color()
+            self.update_effect()
+            await self.update_scene_list()
+
+    @property
+    def device_info(self):
+        """Get device specific attributes."""
+        return {
+            "identifiers": {(DOMAIN, self._mac)},
```

Comment:
```suggestion
            "connections": {(CONNECTION_NETWORK_MAC, self._mac)},
```

`from homeassistant.helpers.device_registry import CONNECTION_NETWORK_MAC`

Reply:
This will endup in duplications in the devices list if an entry was added by the `identifiers`.
Is this change a "hard" requirement? If not, I would stay with the `identifiers` for this PR.

Reply:
Mac addresses are need to be listed as connections and not identifiers.  

The existing entries can be migrated. Example: https://github.com/home-assistant/core/blob/dev/homeassistant/components/tplink/migration.py#L87

Migration is not required.

Reply:
That reminds me, we should change the name space to `wiz` since the vendor also makes smart plugs.

That means everyone will need to set it up again, but from the migration perspective this may be easier as they can run the existing HACS version and then migrate to `wiz` once its merge into core.

Reply:
Jup, sounds clear to me. Thank you. The renaming is on my list ... but I've some fear to do it 😉

Reply:
> but I've some fear to do it 😉

If you get stuck, I'm happy to help clean anything up that goes wrong :)